srcpkgs/mdocml/template

```diff
@@ -30,28 +30,26 @@ post_extract() {
 	# use less(1)
 	sed -i 's,"more -s","less -s",g' main.c
 	sed -i 's,.Xr more,.Xr less,g' *.1
-	
 	sed -i 's/^CC=.*/CC=cc/' configure
 	cat ${FILESDIR}/void.css >>mandoc.css
 	cp ${FILESDIR}/cgi.h .
 }
 pre_configure() {
-	cat >configure.local <<EOF
-PREFIX=/usr
-SBINDIR=/usr/bin
-MANDIR=/usr/share/man
-OSNAME="Void Linux"
-BINM_SOELIM=mandoc-soelim
-MANM_ROFF="mandoc_roff"
-LDFLAGS="$LDFLAGS"
-CFLAGS="$CFLAGS"
-CC="$CC"
-HAVE_REWB_BSD=0
-EOF
+	cat >configure.local <<-EOF
+		PREFIX=/usr
+		SBINDIR=/usr/bin
+		MANDIR=/usr/share/man
+		OSNAME="Void Linux"
+		BINM_SOELIM=mandoc-soelim
+		MANM_ROFF="mandoc_roff"
+		LDFLAGS="$LDFLAGS"
+		CFLAGS="$CFLAGS"
+		CC="$CC"
+		HAVE_REWB_BSD=0
+	EOF
```

Comment:
Why indent this? Why add `-`? Imho this whole block should be kept as is.

Reply:
I just took the chance to make some (mostly) noop changes to the template to make xlint happy. The proposed commit fixes all of the following warnings in the current template:
```
$ xlint srcpkgs/mdocml/template
srcpkgs/mdocml/template: Place checkdepends= after depends=
srcpkgs/mdocml/template: Place provides= after checksum=
srcpkgs/mdocml/template:14: short_desc should not start with an article
srcpkgs/mdocml/template:33: trailing whitespace
srcpkgs/mdocml/template:40: custom variables should use _ prefix: PREFIX=/usr
srcpkgs/mdocml/template:41: custom variables should use _ prefix: SBINDIR=/usr/bin
srcpkgs/mdocml/template:42: custom variables should use _ prefix: MANDIR=/usr/share/man
srcpkgs/mdocml/template:43: custom variables should use _ prefix: OSNAME="Void Linux"
srcpkgs/mdocml/template:44: custom variables should use _ prefix: BINM_SOELIM=mandoc-soelim
srcpkgs/mdocml/template:45: custom variables should use _ prefix: MANM_ROFF="mandoc_roff"
srcpkgs/mdocml/template:49: custom variables should use _ prefix: HAVE_REWB_BSD=0
srcpkgs/mdocml/template:54: trailing whitespace
```

What's the recommended way to fix the warnings for lines 40-49?
The `-` means to strip leading tab characters (https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Here-Documents)

Several templates use this pattern of indented heredoc (for one example see the git template).

I think the indented version is easier to read (either with two tabs or just one tab as in the git template) but I don't really care either way.

Reply:
@gt7-void I believe the point here is that it's ok to fail some `xlint` checks to avoid adding diffs to the templates.

Reply:
> @gt7-void I believe the point here is that it's ok to fail some `xlint` checks to avoid adding diffs to the templates.

I thought it was preferred to fix all `xlint` checks any time a template is changed.

In any case: is there a clear preference for heredocs (indented vs. unindented)?

If unindented heredocs are preferred (or accepted), should `xlint` be able to ignore the heredoc block (or check that the heredoc is in the preferred format)?

Reply:
Could this config simply go in `files/` instead?

Reply:
I needs to expand env vars.